q2_types/citations.bib

```diff
@@ -0,0 +1,20 @@
+@article{mcdonald2012biological,
+  title={The Biological Observation Matrix (BIOM) format or: how I learned to stop worrying and love the ome-ome},
+  author={McDonald, Daniel and Clemente, Jose C and Kuczynski, Justin and Rideout, Jai Ram and Stombaugh, Jesse and Wendel, Doug and Wilke, Andreas and Huse, Susan and Hufnagle, John and Meyer, Folker and others},
```

Comment:
Could we have this include the full author list? Should be: 

```
author={McDonald, Daniel and Clemente, Jose C and Kuczynski, Justin and Rideout, Jai Ram and Stombaugh, Jesse and Wendel, Doug and Wilke, Andreas and Huse, Susan and Hufnagle, John and Meyer, Folker and Knight, Rob and Caporaso, J Gregory},
```

Reply:
oh, it looks like this happened to a couple of these, including the Q1 paper. Looks like google scholar uniformly truncates